openpmd-validator/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "openpmd-validator" %}
+{% set version = "1.1.0.4" %}
+{% set build = 0 %}
+{% set sha256 = "a15153c95350e713866ff44a307548f575bd4a7833b082d65fe2d5d0551d48f8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/openPMD/openPMD-validator/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    # remove checked-in meta.yaml file
+    - rm-meta.patch
```

Comment:
@conda-forge/help-python the check on the source seems to be run before this patch can be applied :´)